app/api.py

```diff
@@ -32,14 +44,12 @@ async def handle_generate(request: Request) -> JSONResponse:
     }
 
     payload = await request.json()
-    payload["prompt"] = "Once upon a time, "  # example interception
+    payload = process_generate_request_hooks(payload)
```

Comment:
Usually at Yandex we named it like this, I think this might be more meaningful:
```suggestion
    json_request = await request.json()
    payload = process_generate_request_hooks(json_request)
```